packs/spell-effects/spell-effect-envenom-companion.json

```diff
@@ -56,6 +56,16 @@
                     "field": "item|system.level.value"
                 }
             },
+            {
+                "key": "Note",
+                "predicate": [
+                    "first-hit",
+                    "unarmed"
+                ],
+                "selector": "strike-damage",
+                "text": "A creature that is damaged by this poison must attempt @Check[fortitude]",
```

Comment:
Please localize this, and the failure condition of the saving throw. Consider adding the poison trait to the check as well.